code-style/git/README.md

```diff
@@ -250,6 +250,24 @@ For the purposes of this example explaining rebasing, let's say that the branch
 
 Voila! You're done. You've successfully rebased a branch onto the master branch!
 
+Naming Branches
+-------------------------
+
+The way we name branches is similar to the style of our commit message. We use types (feat, fix, docs, etc.) to group our branch names so that we know what part of our workflow the branch belongs to combined with a brief subject to describe what the branch does.
+
+**Type**
+
+The allowed types for a branch name are the same as the commit types above. Branch types are followed by two hyphens.
+
+```feat--```
+
+**Subject**
+
+Subjects need to be short, but descriptive, and they may be the same as the subject of your commit. Multiword subjects are separated by a single hyphen in between words.
```

Comment:
Maybe add "message" after "commit" to make it a little more clear so it would read "subject of your commit message..."